### PR TITLE
edit button functionality

### PR DESCRIPTION
**Pull request by mpereannor**
# Description

Added an edit button to a single recipe card. When active, it directly links to edit recipe view page 
## Type of change

Please delete options that are not relevant.

- [ ] New feature (non-breaking change which adds functionality)

## Change status

- [ ] Complete, tested, ready to review and merge

# How Has This Been Tested?
# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code

### CODE REVIEW

File: src/components/EditRecipe/EditRecipeForm/Step1.jsx [hunk 1]
```diff
@@ -193,7 +193,7 @@ function Step1(props) {
       <div>
       <Section1>
       <NavigationSection1>
-            <Link to='/editrecipe/:id'>
+        <Link to={`/editrecipe/${recipeID}`}>
```
Comment (carnundotcom) [inline src/components/EditRecipe/EditRecipeForm/Step1.jsx:196]: Why does there need to be a link to edit within edit steps? (Below too.)

**Review by carnundotcom — CHANGES_REQUESTED** [review]

File: src/components/EditRecipe/EditRecipeForm/Step1.jsx (hunk 1 above)
Comment (RodPaDev) [reply src/components/EditRecipe/EditRecipeForm/Step1.jsx:196]: agreed this doesn't make much sense

Comment (RodPaDev) [reply src/components/EditRecipe/EditRecipeForm/Step1.jsx:196]: You can inject params directly when making a request no need for it to be in history

Comment (mpereannor) [reply src/components/EditRecipe/EditRecipeForm/Step1.jsx:196]: thats inside the back arrow. it redirects to the main edit page

Comment (RodPaDev) [reply src/components/EditRecipe/EditRecipeForm/Step1.jsx:196]: after further investigation, I understand why you're passing in the params and I'd say it's an acceptable strategy to tackle the problem ~but try to find a way without doing it~

**Review by RodPaDev — COMMENTED** [review]
I would like to know how this was tested before approving

**Comment by mpereannor** [comment]
> I would like to know how this was tested before approving

The only way to test it is to run it locally. It worked as planned when i did it though.

File: src/components/EditRecipe/EditRecipeForm/Step1.jsx (hunk 1 above)
Comment (mpereannor) [reply src/components/EditRecipe/EditRecipeForm/Step1.jsx:196]: yeah!! I understand. The forms can't get history from props as its only the container component `EditRecipe`  which is wrapped in a <Route/> the steps component aren't. so it's we use <Link/> or useParams/useHistory

**Review by carnundotcom — APPROVED** [review]

**Review by ThorbenBender — APPROVED** [review]

**Pull request merged**